src/applications/ivc-champva/10-7959C/tests/e2e/fixtures/data/applicant-maximal-test.json

```diff
@@ -1,7 +1,7 @@
 {
   "data": {
     "secondaryAdditionalComments": "Secondary comments",
-    "secondaryMedigapPlan": "medigapPlanC",
+    "secondaryMedigapPlan": "C",
```

Comment:
Changed how Medigap values were referenced. This change was made in all 10-10d E2E JSON files as needed.